config/database.yml.example

```diff
@@ -1,15 +0,0 @@
-development:
-  adapter: postgresql
-  database: rgsocteams_development
-  host: localhost
-
-test:
-  adapter: postgresql
-  database: rgsocteams_test
-  host: localhost
-  min_messages: error
-
-production:
-  adapter: postgresql
-  database: rgsocteams_production
-  host: localhost
```

Comment:
Same as for the other removed file: we don't want this file removed. Can you leave the removal out of this PR please?

Reply:
yeah,sure :)